api/openapi-spec/swagger.json

```diff
@@ -129,60 +129,28 @@
      "operationId": "listVirtualMachineClone",
      "parameters": [
       {
-       "uniqueItems": true,
-       "type": "string",
-       "description": "The continue option should be set when retrieving more results from the server. Since this value is server defined, clients may only use the continue value from a previous query result with identical query parameters (except for the value of continue) and the server may reject a continue value it does not recognize. If the specified continue value is no longer valid whether due to expiration (generally five to fifteen minutes) or a configuration change on the server the server will respond with a 410 ResourceExpired error indicating the client must restart their list without the continue field. This field is not supported when watch is true. Clients may start a watch from the last resourceVersion value returned by the server and not miss any modifications.",
-       "name": "continue",
-       "in": "query"
+       "$ref": "#/parameters/continue-tuthsW5V"
       },
       {
-       "uniqueItems": true,
-       "type": "string",
-       "description": "A selector to restrict the list of returned objects by their fields. Defaults to everything.",
-       "name": "fieldSelector",
-       "in": "query"
+       "$ref": "#/parameters/fieldSelector-xIcQKXFG"
```

Comment:
swagger seems wrong

Reply:
hmm, sure looks odd. I counted about 19 of these. `make apidoc` passes. still checking..

Reply:
I tend to think it's OK. See [here](https://swagger.io/docs/specification/using-ref/) . 
When I look at the output with the IDE, I don't see an issue though admittedly I am no expert.
@oshoval - WDYT? Does the output look OK to you now?

Reply:
If IDE parse it then it sounds good
I am not familiar with it, better to ask someone who does
thanks